articles/openshift/oshift-ref-dns-zone-forwarding.md

```diff
@@ -0,0 +1,68 @@
+---
+title: DNS Zone Forwarding in OpenShift Clusters | UKCloud Ltd
+description: Provides information regarding DNS Zone Forwarding which can be configured in OpenShift clusters to enhance service discovery on deployments with private networks
+services: openshift
+author: Ben Bacon
+reviewer:
+lastreviewed:
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: DNS Zone Forwarding in OpenShift Clusters
+toc_fullpath: Reference/oshift-ref-dns-zone-forwarding.md
+toc_mdlink: oshift-ref-dns-zone-forwarding.md
+---
+
+# DNS Zone Forwarding in OpenShift Clusters
+
+## Overview
+
+This article documents the ability to forward DNS zones within OpenShift deployments. Forwarding zones can assist in resolving services on private networks if you have existing private DNS resolvers configured within your wider UKCloud environment. This removes the need to individually add domains to a node's local host file and can aid consumption of private services from within OpenShift.
+
+### Intended audience
+
+This article is best aimed at those with OpenShift deployments with connectivity to a VRF as other networks commonly have open resolvers available.
+
+## OpenShift Cluster DNS Architecture
```

Comment:
This whole section goes into way too much detail. We don't need to provide the architecture here, just what we can do. The architecture should be in another dedicated kb article if required.
Simply describe the fact that each cluster has DNS servers available that provide DNS forwarding for nodes for each network type.